docs/library/ujson.rst

```diff
@@ -12,10 +12,23 @@ data format.
 Functions
 ---------
 
+.. function:: dump(obj, stream)
+
+   Serialise ``obj`` to a JSON string, writing it to the given ``stream``.
```

Comment:
Markup doesn't follow documentation conventions: https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md#documentation-conventions

Reply:
Thanks, existing docs fixed in 9e8b7b1b635889e365efe3e240c5fe97c0f8de0a